packages/react/src/components/CardEditor/CardEditForm/CardEditFormItems/DataSeriesFormItemModal.jsx

```diff
@@ -515,7 +515,7 @@ const DataSeriesFormItemModal = ({
               onChange={({ selectedItem }) => {
                 if (selectedItem !== mergedI18n.none) {
                   const dataFilter = {
-                    [selectedItem]: availableDimensions[selectedItem][0],
+                    [selectedItem]: availableDimensions[selectedItem].sort()[0],
```

Comment:
Is this to resort the options after selection?

Reply:
it's to select the first option automatically @sstone2423.  I had added sorting to the values before rendering in the list.
I needed to change this because the values were now sorted below and the non-first entry was being selected